vcpkg/get_started/get-started.md

```diff
@@ -149,10 +149,19 @@ This tutorial shows you how to create a C++ "Hello World" program that uses the
 
     Run:
 
+    ::: zone pivot="shell-bash"
     ```console
     cmake --build build
     ```
-
+    ::: zone-end
+    ::: zone pivot="shell-cmd, shell-powershell"
+    ```console
+    .\build\HelloWorld
+    
+    Hello World!
+    ```
+    ::: zone-end
```

Comment:
Maybe I'm looking at a different version of the file? Meant the "Run the application" section.
<img width="455" alt="image" src="https://github.com/microsoft/vcpkg-docs/assets/140731636/2f7182ae-837b-4844-b06d-25e941ca503a">

Reply:
This may be because the new instructions haven't been published but are merged in `main`.

Compare:
* `(live)` https://learn.microsoft.com/en-us/vcpkg/get_started/get-started
* `(main)` https://review.learn.microsoft.com/en-us/vcpkg/get_started/get-started?branch=main

Reply:
That explains it (I don't have perms to access the review link).